nixos/modules/services/web-apps/tt-rss.nix

```
@@ -564,9 +566,12 @@ let
       "Z '${cfg.root}' 0755 ${cfg.user} tt_rss - -"
     ];
 
-    systemd.services.tt-rss =
-      {
+    systemd.services = {
+      phpfpm-tt-rss = mkIf (cfg.pool == "${poolName}") {
+        restartTriggers = [ tt-rss-config tt-rss ];
```

```suggestion
        restartTriggers = [ tt-rss-config pkgs.tt-rss ];
```
sorry for the misleading review